apps/meteor/ee/server/models/raw/LivechatDepartment.ts

```diff
@@ -29,6 +30,10 @@ export class LivechatDepartmentEE extends LivechatDepartmentRaw implements ILive
 		super(db, trash);
 	}
 
+	findByParentId(parentId: string, options?: FindOptions<ILivechatDepartment>): FindCursor<ILivechatDepartment> {
+		return this.col.find({ parentId }, options);
```

Comment:
```suggestion
		return this.find({ parentId }, options);
```

Reply:
This is intentional as we're adding extra filters to the `find` method of this model. So the `col.find` is the unfiltered one